pkg/actionsmetrics/event_reader.go

```diff
@@ -64,6 +64,9 @@ func (reader *EventReader) ProcessWorkflowJobEvent(ctx context.Context, event in
 	runsOn := strings.Join(e.WorkflowJob.Labels, `,`)
 	labels["runs_on"] = runsOn
 	labels["job_name"] = *e.WorkflowJob.Name
+	labels["repository"] = *e.Repo.Name
+	labels["repository_full_name"] = *e.Repo.FullName
+	labels["organization"] = *e.Org.Name
```

Comment:
I was a bit concerned about the increase in cardinality. However, presuming the total number of time series would be `# of repos * # of jobs` anyway, these new labels won't contribute to high cardinality.. (Please correct me if I'm getting it wrong

Reply:
Updated the code, but yes cardinality most probably will increase but not significantly